src/utilities/eventing/Emitter.spec.ts

```diff
@@ -6,11 +6,18 @@ import { configureMockEnvVars } from '../../testUtils/envVars.js';
 import { mockSpy } from '../../testUtils/jest.js';
 import { CE_ID, CE_SOURCE, K_SINK } from '../../testUtils/eventing/stubs.js';
 
+enum CloudEventMode {
+  BINARY = 'binary',
+  STRUCTURED = 'structured',
+  BATCH = 'batch',
```

Comment:
We could skip those two for maintainability reasons. They could be dropped or more could be added in the future, and we don't want to maintain that.